packages-exp/auth-exp/src/platform_browser/persistence/browser.ts

```diff
@@ -73,21 +81,198 @@ function remove(this: BrowserPersistenceClass, key: string): Promise<void> {
 class BrowserLocalPersistence implements BrowserPersistenceClass {
   static type: 'LOCAL' = 'LOCAL';
   type = PersistenceType.LOCAL;
+  storage = localStorage;
   isAvailable = isAvailable;
+
   set = set;
   get = get;
   remove = remove;
-  storage = localStorage;
+
+  // The polling period in case events are not supported
+  static POLLING_TIMER_INTERVAL = 1000;
+  // The IE 10 localStorage cross tab synchronization delay in milliseconds
+  static IE10_LOCAL_STORAGE_SYNC_DELAY = 10;
+
+  private readonly listeners: Record<string, StorageEventListener[]> = {};
```

Comment:
Make this `Record<string, Set<StorageEventListener>>`, then you don't need to do all the splicing / indexOf nonsense in remove

Reply:
done

Reply:
done